app/js/domains.json

```diff
@@ -25,7 +25,9 @@
   {"name": "atwhatsapp", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
   {"name": "atyoutube", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
   {"name": "buymecoffee", "version": "0.9", "registrar": "0xfa6512358d241b636edf5f8ccaaf86676dec21a7"},
+  {"name": "charlielee", "version": "1.0"},
```

Comment:
This seems a little too specialised for general use. Who do you expect to register subdomains of this?

Reply:
It was mostly meant as a joke :) I saw "yourass" on the list and figured it wasn't strictly strict, but if it's not appropriate I completely understand.

Reply:
It's not strict, but I think it has more restricted usefulness than yourass. :)

Reply:
Good argument, you're probably correct :) I thought it might add a bit of flair and I can imagine the current audience might be in agreement, but I definitely agree it doesn't have broad appeal in the same sense as yourass :P

Reply:
If you remove that one, we can merge the pr with the other.

Reply:
Done :)